Sources/Purchasing/StoreKit2/StoreKit2TransactionFetcher.swift

```diff
@@ -307,11 +307,22 @@ extension StoreKit2TransactionFetcher {
                     return nil
                 }
             } catch {
-                self.diagnosticsTracker?.trackAppleAppTransactionError(errorMessage: error.localizedDescription)
-                Logger.warn(Strings.storeKit.sk2_error_fetching_app_transaction(error))
+                trackAppleAppTransactionError(error)
                 return nil
             }
         }
     }
 
+    @available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
+    private func trackAppleAppTransactionError(_ error: Error) {
```

Comment:
I wonder if it would be helpful to know from which of the 2 callers the error happened... but can also be added later if needed

Reply:
hmmm not super useful I think, we just care about the overall amount and types of errors returned by AppTransaction.shared